src/main/docker/disableWeakTlsAlgorithms.patch

```diff
@@ -44,13 +44,7 @@
 +    TLS_RSA_WITH_AES_128_GCM_SHA256, \
 +    TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256, \
 +    TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256, \
-+    TLS_DHE_DSS_WITH_AES_128_GCM_SHA256, \
-+    TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, \
-+    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256, \
-+    TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, \
-+    TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, \
-+    TLS_RSA_WITH_AES_256_GCM_SHA256, \
-+    TLS_DHE_RSA_WITH_AES_256_GCM_SHA256, \
++    TLS_DHE_DSS_WITH_AES_128_GCM_SHA256, \    
```

Comment:
Remove the whitespace from the end of this line.

Reply:
sorry.. done..